language_dump.py

```diff
@@ -1,37 +1,41 @@
 #!/usr/bin/env python3
+"""
+Dump translations for OpenRCT2\'s JSON objects given a language and a file
```

Comment:
The apostrophe doesn't need to be escaped here, right?
```suggestion
Dump translations for OpenRCT2's JSON objects given a language and a file
```